metricflow/model/validations/validator_helpers.py

```diff
@@ -39,82 +40,121 @@ class ModelObjectType(Enum):
     METRIC = "metric"
 
 
+class ValidationContext(BaseModel):
+    """The base context class for vaidation issues"""
+
```

Comment:
We might want to add a Config here that sets the `extra` property to `Extra.forbid` or equivalent. See https://pydantic-docs.helpmanual.io/usage/model_config/#options